test/__init__.py

```diff
@@ -111,23 +121,23 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def onlyBrotli():
+def onlyBrotli() -> Callable[..., Any]:
```

Comment:
Here I think it can be

```py
def onlyBrotli() -> Callable[[_TestFuncT], _TestFuncT]:
```

(again, untested)

Reply:
I've tested this one. seems ok!